Makefile

```diff
@@ -15,48 +15,48 @@ clean:
 	  (./code/gradlew -p code clean)
 
 checkstyle:
-		(./code/gradlew -p code/android-campaignclassic-library checkstyle)
-		
+		(./code/gradlew -p code/campaignclassic checkstyle)
```

Comment:
may be good to add a constant variable at the top of this page to hold your directory for acc.

e.g. `EXTENSION-LIBRARY-FOLDER-NAME = campaignclassic`

then you can use this in all locations instead of hard coding, e.g. 
`(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) checkstyle)`